src/Snippets/one-build.json

```diff
@@ -0,0 +1,76 @@
+{
+    "one-build": {
+        "prefix": "one-build",
+        "body": [
+            "[one-build]",
+            "one-import-tf=${1:True}",
+            "one-import-tflite=${2:False}",
+            "one-import-bcq=${3:False}",
+            "one-import-onnx=${4:False}",
+            "one-optimize=${5:False}",
+            "one-quantize=${6:False}",
+            "one-pack=${7:False}",
+            "one-codegen=${8:False}",
+            "",
+            "[one-import-tf]",
+            "input_path=$9",
+            "output_path=$10",
+            "input_arrays=$11",
+            "input_shapes=$12",
+            "output_arrays=$13",
+            "converter_version=$14",
+            "model_format=$15",
+            "save_intermediate=$16",
+            "",
+            "[one-import-tflite]",
+            "input_path=$17",
+            "output_path=$18",
+            "",
+            "[one-import-bcq]",
+            "input_path=$19",
+            "output_path=$20",
+            "input_arrays=$21",
+            "input_shapes=$22",
+            "output_arrays=$23",
+            "converter_version=$24",
+            "",
+            "[one-import-onnx]",
+            "input_path=$25",
+            "output_path=$26",
+            "input_arrays=$27",
+            "output_arrays=$28",
+            "converter_version=$29",
+            "model_format=$30",
+            "save_intermediate=$31",
+            "",
+            "[one-optimize]",
+            "input_path=$32",
+            "output_path=$33",
+            "",
+            "[one-quantize]",
+            "input_path=$34",
+            "output_path=$35",
+            "input_data=$36",
+            "input_data_format=$37",
+            "generate_profile_data=$38",
+            "input_dtype=$39",
+            "quantized_dtype=$40",
+            "granularity=$41",
+            "min_percentile=$42",
+            "max_percentile=$43",
+            "mode=$44",
+            "",
+            "[one-pack]",
+            "input_path=$45",
+            "output_path=$46",
+            "",
+            "[one-codegen]",
+            "backend=$47",
+            "command=$48",
+            "$0"
```

Comment:
The placeholder should be reordered.

Reply:
I didn't understand the comment.
Can you tell me the problem?

Reply:
@emost22 Oh, I thought the placeholder traversal order is ascending from zero. But, it is from one. Can you let me know what `$0` does?

Reply:
@mhs4670go For $number, tell us the order of the Tab. The position of Tab starts at $1 and $0 means the end of Tab.